mosquitto/DOCS.md

```diff
@@ -78,7 +78,19 @@ A file containing the private key. Place this file in the Home Assistant `ssl` f
 
 ### Option: `require_certificate`
 
-If set to `true` encryption will be enabled using the cert- and keyfile options.
+Whether this option is `true` or `false`:
+- Unencrypted connections are possible on the unencrypted ports (default: `1883`, `1884` for websockets)
+- **If `certfile` and `keyfile` are provided:** encrypted connections are possible on the encrypted ports (default: `8883`, `8884` for websockets) 
+   - In that case, the client must trust the server's certificate
+
```

Comment:
Explaining what `certfile` and `keyfile` is doing here is confusing. We should add this to the relevant options above.

```suggestion
```

If anything, we could just say at the end of this option that `certfile` and `keyfile` are not relevant for the `require_certificate` option, or something along these lines.

Reply:
I updated it, let me know if that works